conf/flight_plans/tudelft/delft_bebop_convergence.xml

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE flight_plan SYSTEM "../flight_plan.dtd">
+
+<flight_plan alt="5" ground_alt="0" lat0="51 59 27.6" lon0="4 22 42.0" max_dist_from_home="200" name="Delft Basic" security_height="2">
+  <header>
+    #include "autopilot.h"
+    #include "subsystems/datalink/datalink.h"
+    #include "subsystems/electrical.h"
+    #include "subsystems/radio_control.h"
+</header>
+  <waypoints>
+    <waypoint name="HOME" x="0.0" y="0.0"/>
+    <waypoint name="CLIMB" x="0.0" y="5.0"/>
+    <waypoint name="STDBY" x="-2.0" y="-5.0"/>
+    <waypoint name="p1" x="-19.9" y="14.3"/>
+    <waypoint name="p2" x="-0.6" y="21.6"/>
+    <waypoint name="p3" x="22.2" y="-26.5"/>
+    <waypoint name="p4" x="4.9" y="-34.4"/>
+    <waypoint name="CAM" x="14.2" y="-29.4"/>
+    <waypoint name="TD" x="5.6" y="-10.9"/>
+    <waypoint name="_HG1" lat="51.99145087814413" lon="4.378416538238525"/>
+    <waypoint name="_HG2" lat="51.99130883618111" lon="4.377756714820861"/>
+    <waypoint name="_HG3" lat="51.99051933891883" lon="4.378344118595123"/>
+    <waypoint name="_HG4" lat="51.99071258535118" lon="4.378966391086578"/>
+    <waypoint name="SG1" lat="51.991269196483096" lon="4.377896189689636"/>
+    <waypoint name="SG2" lat="51.990608529683556" lon="4.378384351730347"/>
+    <waypoint name="SG3" lat="51.990745618845914" lon="4.378829598426819"/>
+    <waypoint name="SG4" lat="51.99136994731342" lon="4.378365576267242"/>
+  </waypoints>
+  <sectors>
+    <sector color="red" name="Flight_Area">
+      <corner name="_HG1"/>
+      <corner name="_HG2"/>
+      <corner name="_HG3"/>
+      <corner name="_HG4"/>
+    </sector>
+    <sector color="yellow" name="Soft_Geo">
+      <corner name="SG1"/>
+      <corner name="SG2"/>
+      <corner name="SG3"/>
+      <corner name="SG4"/>
+    </sector>
+  </sectors>
+  <exceptions>
+    <!--<exception cond="!InsideFlight_Area(GetPosX(), GetPosY())" deroute="Holding point"/>-->
+    <exception cond="Or(!InsideFlight_Area(GetPosX(), GetPosY()), GetPosAlt() > GetAltRef() + 20) &&
```

Comment:
if possible, replace special characters with keywords (e.g `@AND`, `@OR`, `@GT`, `@LT`, ...)